lam4-backend/src/Lam4/Render/Render.hs

```diff
@@ -0,0 +1,100 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Lam4.Render.Render where
+
+import qualified Base.Text                as T
+import           Lam4.Expr.CommonSyntax
+import           Lam4.Expr.ConcreteSyntax
+import           Data.String.Interpolate  (i)
+import qualified Text.Regex.PCRE.Heavy    as PCRE
+--import           Paths_lam4 (getDataFileName)
+
+-- GF-related stuff
+import           Lam4.Render.Lam4Gf
+import           PGF
+
+-- Loosely copied from dsl/…/natural4
+data NLGEnv = NLGEnv
+  { gfGrammar :: PGF
+  , gfLang :: Language
+--  , gfParse :: Type -> T.Text -> [Expr]
+  , gfLin :: PGF.Expr -> T.Text
+  }
+
+gfPath :: String -> String
+gfPath x = [i|gf-grammar/#{x}|]
+
+myNLGEnv :: IO NLGEnv
+myNLGEnv = do
+  -- grammarFile <- getDataFileName $ gfPath "Lam4.pgf"
+  let grammarFile = gfPath "Lam4.pgf"
```

Comment:
A simple but less fragile thing to do in the short term might be to put the path in a variable, as opposed to using string literals. (In the longer term we can think about having a config file etc)

Reply:
Future (i.e. not this PR) TODO for YM: YM will set up some infra for reading in a config with paths down the road. This could be as simple as reading it in from the .env.